src/Microsoft.DotNet.Darc/src/DarcLib/IGitRepo.cs

```diff
@@ -100,7 +107,15 @@ public interface IGitRepo
         /// <param name="pullRequestUrl">Uri of pull request to merge</param>
         /// <param name="parameters">Settings for merge</param>
         /// <returns></returns>
-        Task MergePullRequestAsync(string pullRequestUrl, MergePullRequestParameters parameters);
+     /*   Task MergePullRequestAsync(string pullRequestUrl, MergePullRequestParameters parameters);
```

Comment:
Why comment out this method, but leave it's comments... and include the new comments in this block comment... this probably messed up the xml doc comment parser, since this is an odd format.

Reply:
done